test/CloneFactory.test.js

```diff
@@ -0,0 +1,46 @@
+const SimpleToken = artifacts.require('SimpleToken');
+const SimpleCloneFactory = artifacts.require('SimpleCloneFactory');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('CloneFactory', function ([_, owner]) {
+  let factory;
+
+  describe('factory initialized', function () {
```

Comment:
I would drop this `describe` block for now, no need to make the test suite too complicated.